crates/evm/src/executor/mod.rs

```diff
@@ -778,7 +772,7 @@ fn convert_executed_result(
 
     let result = match out {
         Some(Output::Call(ref data)) => data.to_owned(),
-        _ => Bytes::default(),
+        _ => alloy_primitives::Bytes(Bytes::default()),
```

Comment:
Can unwrap

Reply:
Done in https://github.com/foundry-rs/foundry/pull/5768/commits/91107449b7bbc8e19a70cdb9ffc9489732cb146a